src/web/components/TeamMember/TeamMemberDialog.tsx

```diff
@@ -68,11 +69,11 @@ function TeamMemberDialog(props: TeamMemberDialogProps) {
           <TextInput
             inputName='email'
             label='Email'
-            disabled={!!props.person}
+            disabled={editMode}
             rules={{
               required: 'Please specify email.',
               validate: (value: string) => {
-                if (!validateUniqueTeamMemberEmail(value, props.teamMembers))
+                if (!editMode && !validateUniqueTeamMemberEmail(value, props.teamMembers))
```

Comment:
We don't need to run this validation in edit mode, otherwise we can never actually update a team member as it fails with `Team member email already exists.`